nbconflux/exporter.py

```diff
@@ -82,7 +83,6 @@ def __init__(self, config, **kwargs):
         config.HTMLExporter.preprocessors = [ConfluencePreprocessor]
         config.HTMLExporter.filters = {
             'sanitize_html': sanitize_html,
-            'markdown2html': markdown2html_mistune
```

Comment:
did you mean to delete this?

Reply:
Yes indeed. The base classes adds it to the filter defaults (which I did not realize til now.)